src/components/Api.js

```diff
@@ -0,0 +1,217 @@
+
+const obj = {name: "obbie", ocupation: "Crypto Daytrader"};
+
+class Api {
+  //refactoring needed
+  constructor({ baseUrl, headers }) {
+    this.baseUrl = baseUrl;
+    this.headers = headers;
+  }
+
+  test(){
+    const obj2 = {residance: "USA", hobby: "WallSteet Bets"}
+    obj2.__proto__ = obj
+
+    return obj2.name
+  }
+
+  deleteCard(cardId, handler) {
+    return fetch(`https://around-api.en.tripleten-services.com/v1/cards/${cardId}`, {
```

Comment:
`https://around-api.en.tripleten-services.com/v1`


According to the task you need to pass `baseUrl` into the constructor of `Api`. See here https://disk.yandex.ru/i/J93dbJp7eypXKQ . And in the code you need to use only: `this._baseUrl` (not to duplicate the code)